src/features/game/types/quests.ts

```diff
@@ -56,6 +58,24 @@ export const QUESTS: Record<QuestName, Quest> = {
     wearable: "Farmer Pitchfork",
     deadline: new Date(Date.now() + 10000000000).toISOString(),
   },
+
+  "Chinese New Year Quest 1": {
+    description: "Collect 25 Red Envelopes",
```

Comment:
<img width="197" alt="Screen Shot 2023-01-18 at 10 37 07 am" src="https://user-images.githubusercontent.com/11745561/213035851-5b9f0349-f775-42c8-9430-f37e0610aafb.png">

Is this the intended date? It may be useful to use the end of the event date

Reply:
don't forget to add some padding in the label 😁